### PR TITLE
Fix albumNameEnc can be null in Photos

### LINKED ISSUE #285: AlbumNameEnc error
## The problem
I’m trying to use pyiCloud to download photos to my Ubuntu-based PC but I get an error with AlbumNameEnc

## Environment

- pyiCloud release with the issue (`pip show pyicloud`): 0.9.7
- Last working pyiCloud release (if known): unknown, my script stopped working around Feb 28, 2020
- Service causing this issue: 
- Python version (`python -V`): 3.8
- Operating environment (project deps/Docker/Windows/etc.): Latest Ubuntu, using icloud_photo_downloader as a front-end

## Traceback/Error logs

```shell

pens@gerbil:~/icloud_photos_downloader$ python3 icloudpd.py -d ~/iCloudPhotos --username=Xxxxx --password=Xxxxx
2020-06-30 08:58:19 DEBUG    Authenticating...
Traceback (most recent call last):
  File "icloudpd.py", line 5, in <module>
    main()
  File "/home/pens/.local/lib/python3.8/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/pens/.local/lib/python3.8/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/pens/.local/lib/python3.8/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/pens/.local/lib/python3.8/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/pens/icloud_photos_downloader/icloudpd/base.py", line 276, in main
    photos = icloud.photos.albums[album]
  File "/home/pens/.local/lib/python3.8/site-packages/pyicloud/services/photos.py", line 186, in albums
    folder["fields"]["albumNameEnc"]["value"]
KeyError: 'albumNameEnc'

```

### PR DESCRIPTION
**Pull request by jfburdet**
## Proposed change

Some albums can have null names, which make the current code bash crashing. This changes avoid crash by skipping those kind of albums

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New service (thank you!)
- [ ] New feature (which adds functionality to an existing service)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests
- [ ] Documentation or code sample

## Additional information

- This PR fixes or closes issue: fixes #285

## Checklist

- [x] The code change is tested and works locally.
- [x] Local tests pass. **Your PR cannot be merged unless tests pass**
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated to README

### CODE REVIEW

File: pyicloud/services/photos.py [hunk 1]
```diff
@@ -173,6 +173,11 @@ def albums(self):
             }
 
             for folder in self._fetch_folders():
+
+                # Skiping albums having null name, that can happen sometime
+                if "albumNameEnc" not in folder['fields']:
```
Comment (Quentame) [inline pyicloud/services/photos.py:178]:
```suggestion
                if "albumNameEnc" not in folder["fields"]:
```

To fix Black formating

**Review by Quentame — CHANGES_REQUESTED** [review]
You should fix the CI before we merge

**Comment by fishmanxp** [comment]
I have the same problem, and I don't know how to fix it, can you help me to fix it?
usa635241@163.com
image

**Comment by fishmanxp** [comment]
anyone can help me???

**Pull request merged**